pages/linux/parted.md

```diff
@@ -0,0 +1,36 @@
+# parted
+
+> A partition manipulation program.
+> More information: <https://www.gnu.org/software/parted/parted.html>.
+
+- List partitions on all block devices:
+
+`parted --list`
+
+- Start to manipulate disk partition:
+
+`parted {{/dev/sdX}}`
+
+- Create a new partition table of label-type directly, label-type can be gpt, msdos etc:
+
+`parted --script {{/dev/sdX}} mklabel {{gpt}}`
+
+- Show disk partition information in interactive mode:
+
+`print`
+
+- Select a disk in interactive mode:
+
+`select {{/dev/sdX}}`
+
+- Create a 16GB primary partition with ext4 file type in interactive mode:
+
+`mkpart {{primary}} {{ext4}} {{1}} {{16G}}`
```

Comment:
1. add name (required for GPT).
2. Use parted feature to automatically correct alignment by use of `%`.
```suggestion
`mkpart {{primary}} {{partition_name}} {{ext4}} {{0%}} {{16G}}`
```

Reply:
Great catch!

@einverne, I think you missed this comment :slightly_smiling_face:

Reply:
Updated.